src/bp/ui-studio/src/web/views/OneFlow/diagram/TriggerEditor/index.tsx

```diff
@@ -113,8 +113,8 @@ const EditGoalModal: FC<Props> = props => {
               condition={currentCondition}
               params={currentFlowCondition && currentFlowCondition.params}
               updateParams={onParamsChanged}
+              topicName={topicName}
               contentLang={contentLang}
-              forceSave={forceSave}
```

Comment:
This was set to tell the child component to save itself (eg,when you selected an intent, when you press "save", it added the topicName to the list of the intent's context. We will need a way to set the topic (or categories/context) for the intent

Reply:
I've done that in this PR later on, passing the topicName to the Editor does the trick 🤔

Reply:
Added a debounce that will save the intent's new context when the new one is selected in the dropdown